content/en/docs/setup/best-practices/enforcing-pod-security-standards.md

```diff
@@ -0,0 +1,57 @@
+---
+reviewers:
+- tallclair
+- liggitt
+title: Enforcing Pod Security Standards
+weight: 40
+---
+
+<!-- overview -->
+
+This page provides an overview of best practices when it comes to enforcing [Pod Security Standards](/docs/concepts/security/pod-security-standards).
+
+<!-- body -->
+
+## Using the built-in Pod Security Admission Controller
+
+{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+
+The [Pod Security Admission Controller](/docs/reference/access-authn-authz/admission-controllers/#podsecurity) intends to replace the deprecated PodSecurityPolicies. 
+
+### Configure all cluster namespaces
+
+Even if the initial configurations are most permissive (i.e. everything is `privileged`), any Namespaces that lack any configuration at all will become significant gaps in your cluster security model.
```

Comment:
I'm not sure I agree with labeling all namespaces as `privileged` as a starting point... if I were migrating a cluster to this, I'd use unlabeled namespaces to represent "not yet evaluated", distinct from "evaluated and explicitly granted `privileged`"